update/repository_test.go

```diff
@@ -0,0 +1,291 @@
+package update_test
+
+import (
+	"testing"
+
+	"fmt"
+	"github.com/fabric8-services/fabric8-tenant/test"
+	"github.com/fabric8-services/fabric8-tenant/test/doubles"
+	"github.com/fabric8-services/fabric8-tenant/test/gormsupport"
+	"github.com/fabric8-services/fabric8-tenant/test/resource"
+	"github.com/fabric8-services/fabric8-tenant/update"
+	"github.com/jinzhu/gorm"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"time"
+)
+
+type UpdateRepoTestSuite struct {
+	gormsupport.DBTestSuite
+}
+
+func TestUpdateRepository(t *testing.T) {
+	resource.Require(t, resource.Database)
+	suite.Run(t, &UpdateRepoTestSuite{DBTestSuite: gormsupport.NewDBTestSuite("../config.yaml")})
+}
+
+func (s *UpdateRepoTestSuite) TestUpdateAndGetStatus() {
+	s.T().Run("set and get status should pass", func(t *testing.T) {
+		// given
+		err := update.Transaction(s.DB, func(tx *gorm.DB) error {
+			return update.NewRepository(tx).UpdateStatus(update.Updating)
+		})
+		require.NoError(s.T(), err)
```

Comment:
use `require.NoError(t, err)` otherwise the test execution will crash with a `panic` in case of error

Reply:
wrong copy-paste :-) a64525f